@xen-orchestra/web/src/stores/xo-rest-api/pif.store.ts

```diff
@@ -0,0 +1,28 @@
+import type { XoHost } from '@/types/xo/host.type'
+import type { XoPif } from '@/types/xo/pif.type'
+import { createXoStoreConfig } from '@/utils/create-xo-store-config.util'
+import { createSubscribableStoreContext } from '@core/utils/create-subscribable-store-context.util'
+import { defineStore } from 'pinia'
+import { computed } from 'vue'
+
+export const usePifStore = defineStore('pif', () => {
+  const { context: baseContext, ...config } = createXoStoreConfig('pif', {})
```

Comment:
Prefer `configRest` over `config` to make clearer that the object is incomplete.

```suggestion
  const { context: baseContext, ...restConfig } = createXoStoreConfig('pif')

```